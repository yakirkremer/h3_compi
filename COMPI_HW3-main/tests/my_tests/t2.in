int foo(){
    return 5;
}

byte g(){
    return 5b;
}

bool fun(){ return true;}

void main(){
    auto a = foo();
    auto c = g();
    auto d = fun();
    auto e = false;
    a = foo();
    c = g();
    d = false;
    a = g();
    a = (int) g();
    a = (byte) g();
    c = (byte) foo();
    a = d;
}