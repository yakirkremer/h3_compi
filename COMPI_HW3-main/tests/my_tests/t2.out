---end scope---
---end scope---
---end scope---
line 23: type mismatch
